win32ss/user/user32/misc/exticon.c

```diff
@@ -714,9 +714,78 @@ static UINT ICO_ExtractIconExW(
 	    xresdir = find_entry_by_id(iconresdir, LOWORD(pIconId[i]), rootresdir);
             if( !xresdir )
             {
+#ifdef __REACTOS__
+              /* XP/2K3 can decode icons this way. Vista/Win7, cannot. This
+               * handles damaged Resources in 'Icon Group' by using the 'Icon'
+               * resources. Error can occur in older Watcom C/C++ compiles. */
+              const IMAGE_RESOURCE_DIRECTORY *resdir;
+              WARN("icon entry %d not found\n", LOWORD(pIconId[i]));
```

Comment:
It would only be necessary to someone to look at the source of the WARN in the code and read the comments above to know this. So I consider it as not needed. Thanks.